src/components/Loader.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useRef } from "react";
+import { usePromiseTracker } from "react-promise-tracker";
+import Loading from "react-loader-spinner";
+import { useSelector } from "react-redux";
+import { selectShowHeader } from "../store/reducers/movies";
+
+type Props = {
+    area?: string
+    type?: "Audio"
+        |"BallTriangle"
+        |"Bars"
+        |"Circles"
+        |"Grid"
+        |"Hearts"
+        |"Oval"
+        |"Puff"
+        |"Rings"
+        |"TailSpin"
+        |"ThreeDots"
+        |"Watch"
+        |"RevolvingDot"
+        |"Triangle"
+        |"Plane"
+        |"MutatingDots"
+        |"CradleLoader"
+    color?: string
+    height?: string
+    width?: string
+}
+
+const Loader:React.FC<Props> = ({
+    area, type, color, height, width,
+}) => {
```

Comment:
Not an error, but instead of `||` you can define default props like this:
```js
{area, type = 'Circles', color = '#555555', height = 400, width = 400}
```

Reply:
fixed it)